Tests/FeaturevisorSDKTests/InstanceTests+Variables.swift

```diff
@@ -0,0 +1,87 @@
+import Foundation
+import XCTest
+
+@testable import FeaturevisorSDK
+@testable import FeaturevisorTypes
+
+extension FeaturevisorInstanceTests {
```

Comment:
In my opinion is better to have separate test class `FeaturevisorInstanceVariablesTests` instead of extension, because sometimes Xcode don't mark tests in extensions correctly - but's just a minor

Reply:
Yessss, agree. Already changed this in another PR. Hope you don't mind ;)